### PR TITLE
Warp detection

### PR DESCRIPTION
**Pull request by dencelle**
this should be a fairly light weight way of detecting warps.
since its not distance dependent like we've always done in the past, we shouldn't get any false positives.

### CODE REVIEW

**Comment by mackal** [comment]
I don't think we need the worldserver.SendEmoteMessage

File: zone/client_packet.cpp [hunk 1]
```diff
@@ -5865,6 +5865,30 @@ void Client::Handle_OP_Fishing(const EQApplicationPacket *app)
 	// forage for.
 }
 
+void Client::Handle_OP_FloatListThing(const EQApplicationPacket *app) {
+	UpdateMovementEntry* m_MovementHistory = new UpdateMovementEntry[(app->size) / sizeof(UpdateMovementEntry)];
```
Comment (mackal) [inline zone/client_packet.cpp]: This leaks on the line below

File: zone/client_packet.cpp [hunk 2]
```diff
@@ -5865,6 +5865,30 @@ void Client::Handle_OP_Fishing(const EQApplicationPacket *app)
...
+	m_MovementHistory = (UpdateMovementEntry*)app->pBuffer;
+	for (int index = 0; index < (app->size) / sizeof(UpdateMovementEntry); index++) {
+		switch (m_MovementHistory[index].type) {
+		case 3:
```
Comment (mackal) [inline zone/client_packet.cpp]: Can we make an enum instead of magic numbers?

File: common/eq_packet_structs.h [hunk 3]
```diff
@@ -5418,6 +5418,15 @@ struct SayLinkBodyFrame_Struct {
 /*056*/
 };
 
+struct UpdateMovementEntry {
+	/* 00 */	float Y;
+	/* 04 */	float X;
+	/* 08 */	float Z;
+	/* 12 */	byte type;
```
Comment (mackal) [inline common/eq_packet_structs.h]: We usually just use uint8 for this, can that be changed for consistency?

**Comment by Akkadius** [comment]
Closing because I know this was not complete and there were other efforts in progress

**Pull request closed without merging**